### PR TITLE
[Docs] Add AdminApi docs to bundle

### PR DESCRIPTION
**Pull request by Tomanhez**

### CODE REVIEW

**Review by lchrusciel — COMMENTED** [review]
Shouldn't we translate these docs to markdown?

**Review by SirDomin — APPROVED** [review]

**Review by GSadee — DISMISSED** [review]
These docs should be translated to markdown and it would be nice to mention about them in README file

File: translator.php [hunk 1]
```diff
@@ -0,0 +1 @@
+<?php
```
Comment (Zales0123) [inline translator.php]: 💃 🐎 🚀

File: Controller/ProductTaxonPositionController.php [hunk 2]
```diff
@@ -45,6 +45,7 @@ public function updatePositionsAction(Request $request): JsonResponse
             return new JsonResponse(null, Response::HTTP_NO_CONTENT);
         }
 
+        /** @psalm-suppress InvalidIterator */
```
Comment (Zales0123) [inline Controller/ProductTaxonPositionController.php:48]: I suppose it's required to fix the build?

**Review by Zales0123 — APPROVED** [review]
I don't think it's possible to review this PR properly 😄 I believe it's working, we would be fixing bugs if any occurs 🖖

**Comment by Zales0123** [comment]
One more thing - [as Grzegorz said](https://github.com/Sylius/SyliusAdminApiBundle/pull/6#pullrequestreview-691338798), we should link this documentation in the README file

File: README.md [hunk 3]
```diff
@@ -15,6 +15,11 @@ Documentation
 
 Documentation is available on [**docs.sylius.com**](https://docs.sylius.com/en/latest/api/index.html).
 
+AdminApiBundle
+--------------
+
+AdminApiBundle [Documentation](docs/index.md)
```
Comment (Zales0123) [inline README.md]: I believe we should just change the previous section link :)

Comment (SirDomin) [reply README.md]: :)

**Review by GSadee — APPROVED** [review]

**Comment by GSadee** [comment]
Thank you, Tomasz! :1st_place_medal:

**Pull request merged**